backends/dotnet/src/wpf/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ elseif(CMAKE_SIZEOF_VOID_P EQUAL 4)
 	set(CMAKE_CSharp_FLAGS "/platform:x86")
 endif()
 
-set(CMAKE_DOTNET_TARGET_FRAMEWORK_VERSION "v4.0")
+set(CMAKE_DOTNET_TARGET_FRAMEWORK_VERSION "v4.8")
```

Comment:
Why did you increase the target .NET version?  
In this case a subset of programs built with .NET Framework versions 4.0-4.8 will not be supported by this backend.

Reply:
While debugging, I couldn't use .NET Target Framework v4.0 (I already have v 4.8. and .NET doesn't let me roll back the Framework version).
It will be fixed in the next commits.